index.html

```diff
@@ -1197,6 +1201,22 @@
 
 The full representation in code of this linear combination of BRDF/BTDF/BSSRDF lobes will depend on renderer-specific details.
 
+### Entering versus exiting
+
+As aspect that has been ignored in this approximation (in the non-thin-walled case) is the different light transport for rays incident from the top or the bottom side. In reality the physical effect of the layers differs in these cases. For entering rays:
+
+   - The fuzz reflection is un-tinted by the coat absorption.
```

Comment:
```suggestion
   - The fuzz reflection is not tinted by the coat absorption.
```
("un-tinted" sounds to me like a tint is removed somehow)